src/main/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshService.java

```diff
@@ -36,38 +36,36 @@ public MeshService(MeshClient meshClient,
 
     @Scheduled(fixedRateString = "${nhais.mesh.scanMailboxIntervalInMilliseconds}")
     public void scanMeshInboxForMessages() {
-        LOGGER.debug("Trying to scan MESH mailbox");
         if (!meshMailBoxScheduler.isEnabled()){
-            LOGGER.warn("MESH mailbox scheduler is disabled. Set proper env var to enable it");
+            LOGGER.warn("Not running the MESH mailbox polling cycle because it is disabled. Set variable " +
+                "NHAIS_SCHEDULER_ENABLED to true to enable it.");
             return;
         }
+        LOGGER.info("Requesting lock from database to run MESH mailbox polling cycle");
         if (meshMailBoxScheduler.hasTimePassed(scanDelayInSeconds)) {
-            LOGGER.info("Mesh messages scan started");
-            List<String> inboxMessageIds = downloadMessageIds();
+            LOGGER.info("Starting MESH mailbox polling cycle");
+            List<String> inboxMessageIds = meshClient.getInboxMessageIds();
+            LOGGER.info("There are {} messages in the MESH mailbox", inboxMessageIds.size());
             for (String messageId : inboxMessageIds) {
                 try {
+                    LOGGER.debug("Downloading message id {}", messageId);
                     InboundMeshMessage meshMessage = meshClient.getEdifactMessage(messageId);
+                    LOGGER.debug("Publishing content of message id {} to inbound mesh MQ", messageId);
```

Comment:
Since it's DEBUG, it might be worth logging content as well

Reply:
comment on the next line explains this

Reply:
One for the publisher component. I'm only working on the mesh client side right now.